syncer/syncer.go

```diff
@@ -154,6 +154,7 @@ type Syncer struct {
 		sync.RWMutex
 		t time.Time
 	}
+	safeMode *sm.SafeMode
```

Comment:
Can we add a comment here to explain why this variable must be a local variable?

Reply:
So I think we can add a comment for the `safeMode` in eventContext?

Reply:
That's also fine. To avoid this line is changed in the future.